src/components/book/book.css

```diff
@@ -55,7 +43,7 @@ article {
 }
 
 .Text-Style-6 {
-  font-family: RobotoSlab, serif;
+  font-family: "Roboto Slab", serif;
   font-size: 1.375rem;
   font-weight: bold;
   font-stretch: normal;
```

Comment:
Great job @jojo987N 🤝

- [ ] Just like the previous reviewer suggested, kindly remove the repeating lines and assign one class attribute for multiple elements with the same rules. You may work on the following lines, `"Roboto Slab", serif;`, `"Montserrat", sans-serif;`, etc.